src/aiori-HDF5.c

```diff
@@ -235,6 +293,18 @@ static void *HDF5_Open(char *testFileName, IOR_param_t * param)
         accessPropList = H5Pcreate(H5P_FILE_ACCESS);
         HDF5_CHECK(accessPropList, "cannot create file access property list");
 
+        /* Get the function for the property of DAOS object class */
+        if((H5daos_set_object_class = dlsym(dl_handle, "H5daos_set_object_class")) == NULL) {
```

Comment:
why is this dynamically loaded? Isn't there a function defined in the API for this?

Reply:
This version of IOR doesn't link to the DAOS-VOL library.  It uses the DAOS-VOL as the connector, which is loaded dynamically.  So I can call HDF5 functions because the HDF5 library is linked.  But I cannot call any H5daos_xxx function directly.

Reply:
that's not great.. this is the opposite of saying we need no application changes.
this is not related to your work, so we can discuss it during a different meeting.